commcare_export/utils.py

```diff
@@ -50,8 +51,9 @@ def print_runs(runs):
 
     StreamingMarkdownTableWriter(
         sys.stdout, compute_widths=True
-    ).write_table({
```

Comment:
was this just not working before?

Reply:
It was written in 2018, and I think the parameter type was changed from a dict to a TableSpec in 2020. Currently I get

```
$ python commcare_export/utils_cli.py history \
     --query example_det_config.xlsx \
     --project demo \
     --output postgresql://postgres:postgres@localhost:5432/postgres

Listing checkpoints (most recent 10):
    project:        demo
    query filename: example_det_config.xlsx

Traceback (most recent call last):
  File "commcare_export/utils_cli.py", line 167, in <module>
    entry_point()
  File "commcare_export/utils_cli.py", line 163, in entry_point
    main(sys.argv[1:])
  File "commcare_export/utils_cli.py", line 154, in main
    exit(main_with_args(args))
  File "commcare_export/utils_cli.py", line 159, in main_with_args
    command().run(args)
  File "commcare_export/utils_cli.py", line 62, in run
    print_runs(runs)
  File "commcare_export/utils.py", line 51, in print_runs
    StreamingMarkdownTableWriter(
  File "commcare_export/writers.py", line 233, in write_table
    col_widths = self._get_column_widths(table)
  File "commcare_export/writers.py", line 259, in _get_column_widths
    all_rows = [table.headings] + table.rows
AttributeError: 'dict' object has no attribute 'headings'

```

Reply:
Yeah, the jury is still out wrt to type hints in HQ. But the type hints in commcare-export have been here for 9 years. I just made them parseable, and fixed them. And then fixed the bug that the now-parseable type hints exposed.

If this seems like a step backwards, we can revert this PR -- it's why I moved the typing stuff into its own branch But to me it feels like I made some unhelpful docstrings a bit more helpful.